spec/lib/packet.spec.js

```diff
@@ -311,6 +343,209 @@ describe("Packet", function() {
     });
   });
 
+  describe("#parseResponseData", function() {
+    var payload;
+
+    beforeEach(function() {
+      payload = {
+        sop1: 0xFF,
+        sop2: 0xFF,
+        mrsp: 0x00,
+        seq: 0x02,
+        dlen: 0x01,
+        data: new Buffer(0),
+        checksum: 0xFC,
+      };
+
+      stub(packet, "_parseData");
+      packet._parseData.returns({ val1: "uno" });
+
+      packet.parseResponseData({ did: 0x02, cid: 0x07 }, payload);
+    });
+
+    it("returns payload if cmd is not valid", function() {
+      spy(packet, "parseResponseData");
+      packet.parseResponseData({}, payload);
+      expect(packet.parseResponseData).to.returned(payload);
```

Comment:
I'm unsure what's happening in this test, can you explain? It doesn't look like there's any reason to spy.